File: week3/nodejs-basics/pattern.js
const readline=require("readline-sync")
const limit=Number.parseInt(readline.question("enter the limit"))
function getinitialParameter(limit){
    let initialparameter=(limit * (limit+1) )/2;
    return initialparameter
}
for(let i=limit;i>=1;i--){
    let initialparameter=getinitialParameter(i)
    for(let j=i;j>=1;j--){
        process.stdout.write(initialparameter-j+1+'  ')

    }
    console.log()
}

File: week3/nodejs-basics/pattern.py
limit=int(input("enter the limit"))
def getinitialParameter(limit):
    initialparameter=int((limit * (limit+1)  )/2)
    return initialparameter

for i in range(limit,0,-1):
    initialparameter=getinitialParameter(i)
    for j in range(i,0,-1):
        print(initialparameter-j+1,end=" ")
    print("\n")

